packages/example/src/modules/program/view/ProgramListVariableSize.tsx

```diff
@@ -0,0 +1,82 @@
+import styled from '@emotion/native';
+import { useTheme } from '@emotion/react';
+import { useNavigation } from '@react-navigation/native';
+import { NativeStackNavigationProp } from '@react-navigation/native-stack';
+import { useCallback, useMemo } from 'react';
+import { SpatialNavigationVirtualizedList } from 'react-tv-space-navigation';
+import { RootStackParamList } from '../../../../App';
+import { ProgramNode } from './ProgramNode';
+import { scaledPixels } from '../../../design-system/helpers/scaledPixels';
+import { ProgramNodeLandscape } from './ProgramNodeLandscape';
+import { getPrograms } from '../infra/programInfos';
+import { ProgramInfo } from '../domain/programInfo';
+
+const NUMBER_OF_ITEMS_VISIBLE_ON_SCREEN = 7;
+const WINDOW_SIZE = NUMBER_OF_ITEMS_VISIBLE_ON_SCREEN + 8;
+const ROW_PADDING = scaledPixels(70);
+
+export const ProgramList = ({
+  orientation,
+  containerStyle,
+}: {
+  orientation?: 'vertical' | 'horizontal';
+  containerStyle?: object;
+}) => {
+  const navigation = useNavigation<NativeStackNavigationProp<RootStackParamList>>();
+
+  const renderItem = useCallback(
+    ({ item }: { item: ProgramInfo }) =>
+      parseInt(item.id, 10) % 2 === 0 ? (
```

Comment:
You could take this out into a local function, that'd be a little simpler to read I think

```suggestion
      isLargeItem(item.id) ? (
```

Reply:
Done :)